README.md

```diff
@@ -1,12 +1,12 @@
 # Databricks SDK for Go
 
-[Beta](https://docs.databricks.com/release-notes/release-types.html): This SDK is supported for production use cases, 
-but we do expect future releases to have some interface changes; see [Interface stability](#interface-stability). 
-We are keen to hear feedback from you on these SDKs. Please [file issues](https://github.com/databricks/databricks-sdk-go/issues), and we will address them 
+[Beta](https://docs.databricks.com/release-notes/release-types.html): This SDK is supported for production use cases,
+but we do expect future releases to have some interface changes; see [Interface stability](#interface-stability).
```

Comment:
nit: `changes; see`

Instead of `;` it probably should be `,`

Reply:
That's probably true. These changes are all from VSCode's markdown formatter, so I didn't look very closely at this.